lookout/style/format/rules.py

```diff
@@ -207,6 +227,8 @@ def append_new_rule(labels: Tuple[int, ...], y_i: int, conf: float, support: int
         y_indices = {id(vnode): i for i, vnode in enumerate(vnodes_y)}
         for vnode1, vnode2, vnode3 in zip(vnodes, islice(vnodes, 1, None),
                                           islice(vnodes, 2, None)):
+            # What if we have prediction for one quote label and do not have for another?
+            # Should we change both or should we keep original?
```

Comment:
It can happen because of this lines:
https://github.com/src-d/style-analyzer/pull/478/files#diff-fd9a5210475c52f533d210a1888cffa5R134

I think it is a question for Hugo. 
I will delete a comment when we resolve this situation.

Reply:
Does still hold after we merged #482?

Reply:
not sure. Let's wait for @m09 comment.

Reply:
I think we should still pass it to harmonize quotes. I'll push with the visualizer fix

Reply:
Ok so after having observed the current result in visualizer I think this works fine (if we have prediction for only one quote it will undergo the uast breaking check anyway so if it's breaking it'll get discarded, and this way it can still fix whitespace, newlines, …)